browser/src/Platform.ts

```diff
@@ -1,8 +1,40 @@
+/// <reference path="./../../definitions/sudo-prompt.d.ts"/>
+
+import * as fs from "fs"
 import * as os from "os"
+import * as sudo from "sudo-prompt"
 
 export const isWindows = () => os.platform() === "win32"
 export const isMac = () => os.platform() === "darwin"
 export const isLinux = () => os.platform() === "linux"
 
 export const getUserHome = () =>
     isWindows() ? process.env["USERPROFILE"] : process.env["HOME"] // tslint:disable-line no-string-literal
+
+export const getLinkPath = () => isMac() ? "/usr/local/bin/oni" : "" // TODO: windows + linux
+export const isAddedToPath = () => {
+  if (isMac()) {
+    try { fs.lstatSync(getLinkPath()) } catch (_) { return false }
+    return true
+  }
+
+  return false
+}
+export const removeFromPath = () => isMac () ? fs.unlinkSync(getLinkPath()) : false // TODO: windows + other
+
+export const addToPath = async () => {
+  if (isMac()) {
+    const appDirectory = "/Applications/Oni.app/Contents/"
```

Comment:
Can we derive this from `process.execPath` or `process.helperExecPath`? This logic will work fine in the install-from-dmg case, but I see a decent amount of .zip downloads too. It'd be great if it worked in that case as well.